examples/example-myst/myst.yml

```diff
@@ -17,18 +17,18 @@ project:
   bibliography: []
   exports:
     - format: typst
-      template: typst-thesis.typ
+      template: ../../thesis-template
```

Comment:
This has to be the folder, not a file. It is picking up the folder with the template.yml in it.

Reply:
Aaaaah, that makes sense! No idea that was the case!